### ISSUE TITLE
The questionnaire is not displaying the proper hint for correct answers

### ISSUE THREAD

**Issue by jaguarfi** [apluslms/mooc-grader#104]
Hints for correct answers are not properly displayed. 

```rst
  .. pick-one:: 10
    :required:

    **Modifiers used in this question:** ``required``.

    When :math:`(x + 1)^3 = 27`, what is :math:`x`?

    a. 9
    *b. 2
    c. 3

    a § Not quite. Remember the cube root.
    b § This is correct
    c § Rather close. Remember that you can add or subtract the same number to both sides of the equation.
```
In the previous exercise, if the user chooses the option `b.`, Aplus must display the **"This is correct"** text. But, in v1.8 Aplus is only showing the default text => **"Correct"**

*Aplus v1.7* ***working example***
![working](https://user-images.githubusercontent.com/10437475/107642425-8996b800-6c7d-11eb-8c97-922df0a8c07a.png)

*Aplus v1.8* ***non-working example***
![non-working](https://user-images.githubusercontent.com/10437475/107642680-dc706f80-6c7d-11eb-992e-c9994d7b9b59.png)

**Comment by Mankro** [comment]
This may be the code that skips the hint for correct answers: https://github.com/apluslms/mooc-grader/blob/6e5bc6b177debefcb404bbf4b0c2f77715e9cf2a/access/templates/access/graded_form_feedback.html#L26

**Comment by jaguarfi** [comment]
It seems that `pick-any` directives still work. The problem is presented only in the `pick-one`

**Comment by skulonen** [comment]
@jaguarfi said that `pick-any` questions work correctly. I just tested them, and I'm not so sure about that. It seems that `pick-any` questions only display hints for options that haven't been selected. For example, in `aplus-manual`, the question "When (x+1)2=16, what is x?" has the following option:

`d. an irrational number`

Which should display the following hint when selected:

`d § No. This equation has a nice and easy solution.`

But that didn't work when I tested it. Correct answer hints didn't seem to work either (i.e. when a correct option is selected). So, should we also fix `pick-any` hints on this issue?
